checker/examples/GradleExamples/GradleJava7Example/build.gradle

```diff
@@ -0,0 +1,60 @@
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+
+
+repositories {
+    mavenCentral()
+}
+
+if (JavaVersion.current().isJava8Compatible()) {
+    throw new GradleException("This example requires Java 7, but you are using Java 8. Please see the Java 8 example in ../GradleJava8Example")
+}
+
+assert JavaVersion.current().isJava7()
```

Comment:
Thanks for the explanation.  I didn't realize that the key distinction was whether annotations are in comments.

I suggest that the Gradle build files be merged, the example source code not be merged (but be identical except for the comment markers around annotations), and the instructions and possibly the directory names clarify the annotations-in-comments distinction.